_1_basics/src/main/java/code/_3_in_class/Main.java

```diff
@@ -1,8 +1,34 @@
 package code._3_in_class;
 
 public class Main {
-
   public static void main(String[] args) {
-    //TODO put your code changes in here
+    fizz1();
+  }
+
+  public static void fizz1() {
```

Comment:
1. te rog sa creezi o clasa noua FizzBuzz in pachetul "_4_student_effort"
2. clasa Main de aici te rog sa faci undo la toate modificarile si sa o lasi goala asa cum era initial in github
3. te rog sa creezi o clasa nou in pachedul "_3_in_class" cu numele "HelloWorld" care sa afiseze la consola cuvintele "hello world"
4. te rog sa modifici in github numele pull request-ului tau sa contina doar litere mari

Reply:
nu gasesc pe lista de studenti admisi la curs numele "Teodor Ganga". 
te rog sa iei legatura cu cei din organizare si sa rezolvati si problema asta ...